database/migrations/2016_04_27_024324_create_register_confirms_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
So as mentioned before, i don't see any reason to create one more table for 1 - to - 1 relation.